cirq/google/calibration/workflow.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from cirq.google.calibration.phased_fsim import (
+    PhasedFSimCalibrationRequest,
+    PhasedFSimCalibrationResult
+)
+from cirq.google.engine import Engine
+
+
+def run_characterizations(calibrations: List[PhasedFSimCalibrationRequest],
+                          engine: Engine,
+                          processor_id: str,
+                          handler_name: str
+                          ) -> List[PhasedFSimCalibrationResult]:
```

Comment:
Add docstring and tests for functions in this file.

Reply:
Done